include/runtime-support.h

```diff
@@ -268,6 +268,11 @@ std::string file_begin(const std::optional<std::string>& mime_type);
  */
 std::string fuid();
 
+/**
+ * Close the current active connection.
+ */
+void close_connection();
```

Comment:
How about calling this `terminate_session` to go with the more general terminology that the session manager brings these days?